.github/workflows/test_thorough.sh

```diff
@@ -7,6 +7,8 @@ grass79 --tmp-location XY --exec \
     g.extension g.download.location
 grass79 --tmp-location XY --exec \
     g.download.location url=http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha2.tar.gz dbase=$HOME
+grass79 $HOME/nc_spm_full_v2alpha2/PERMANENT --exec \
+    t.upgrade
```

Comment:
Is there anything in the doc or code which would suggest that state of db in the PERMANENT mapset has any effect on the newly created mapsets or other mapsets in general? It also seems to be unrelated to what remains of #1342 based on the runs and https://github.com/OSGeo/grass/issues/1342#issuecomment-795553590 by @veroandreo. If there is no reason to have this (and the other) line, let's remove it (them) from this PR.

Reply:
IIUC, and from what I have tested, t.upgrade is only needed when you have old temporal db in your mapsets. 

If tests for grass-dev run every time from scratch in mapsets without any temporal db, the t.upgrade step is not needed because the temporal database will be version 3 then, nothing to upgrade.

Reply:
Unfortunately `PERMANENT` mapset in `nc_spm_full_v2alpha2` already contains TGIS database.

```
ls ~/grassdata/nc_spm_full_v2alpha2/PERMANENT/tgis
sqlite.db
```

But the connection is not defined:

```
grass79 ~/grassdata/nc_spm_full_v2alpha2/PERMANENT --exec t.connect -p

driver:
database:
```

This is **inconsistent**: the TGIS database exists, but the connection is not defined.

The TGIS database is in version 2, this is a reason why it must be upgraded by `t.upgrade`.

```
grass79 ~/grassdata/nc_spm_full_v2alpha2/PERMANENT --exec t.info -d

Default TGIS driver / database set to:
driver: sqlite
database: $GISDBASE/$LOCATION_NAME/$MAPSET/tgis/sqlite.db
ERROR: Unsupported temporal database: version mismatch.
The format of your actual temporal database is not supported any more.
Please create a backup of your temporal database to avoid lossing data.
SOLUTION: Run t.upgrade command installed from GRASS Addons in order to
upgrade your temporal database.
Supported temporal database version is: 2
Current temporal database info:
DBMI interface:..... sqlite3
Temporal database:..
/home/martin/grassdata/nc_spm_full_v2alpha2/PERMANENT/tgis/sqlite.db
```

@wenzeslaus The correct solution seems to be to remove `tgis` directory from http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha2.tar.gz

Reply:
@vero is right. When creating a new mapset, TGIS database located in `PERMANENT` mapset has no relation to the newly created TGIS database. 

```
grass79 -c ~/grassdata/nc_spm_full_v2alpha2/user2 --exec t.info -d
...
 | tgis_db_version .......... 3
```

Reply:
Although it is nice to know I was right from the beginning :-) that the sample dataset doesn't have anything to do with the failing tests, it is now important to ask even more than before: Is the mapset database independence documented or not? What is in the code?